amqp_job_test.go

```diff
@@ -196,7 +197,7 @@ func TestAMQPJob_createStateUpdateBody(t *testing.T) {
 	job.received = time.Time{}
 	assert.NotContains(t, job.createStateUpdateBody("foo"), "received_at")
 
-	job.Payload().Job.QueuedAt = &time.Time{}
+	job.Payload().Job.QueuedAt = nil
```

Comment:
Hi @meatballhat I understood everything else, but this. 
Why `nil` ?  Because we're moving up the `QueuedAt` and assigning it to time.Time up at #L53?